tests/test_bzutil.py

```diff
@@ -421,6 +421,25 @@ def test_is_first_fix_any_validate(self):
         actual = bzutil.is_first_fix_any(bzapi, bug, tr)
         self.assertEqual(expected, actual)
 
+    def test_sort_cve_bugs(self):
+        flaw_bugs = [
+            flexmock(alias=['CVE-2022-123'], severity='midium'),
+            flexmock(alias=['CVE-2022-456'], severity='urgent'),
+            flexmock(alias=['CVE-2021-789'], severity='midium'),
+            flexmock(alias=['CVE-2021-100'], severity='midium')
+        ]
+        sort_list = bzutil.sort_cve_bugs(flaw_bugs)
+        expected = [
```

Comment:
There's a typo: `midium` is not expected. I think `Low` is an impact, are you sure these words are capitalized correctly?
Sorting should happen first on Severity, and then on Alias alphabetically. Think the test does not reflect that.

Reply:
severity have different value in bugzilla and jira, and we use function `constants.security_impact_map[get_highest_security_impact([bug])] `o translate it to integer so they can compare, this test reflect the severity value was sorted and cve number also sorted

Reply:
Is it `medium`?